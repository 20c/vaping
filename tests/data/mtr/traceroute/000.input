traceroute to 2.2.141.1 (2.2.141.1), 30 hops max, 60 byte packets
 1  204.130.133.130  0.241 ms
 2  38.122.209.113  0.925 ms
 3  154.24.55.105  1.459 ms
 4  154.54.0.225  4.312 ms
 5  154.54.45.18  2.036 ms
 6  154.54.10.34  4.844 ms
 7  *
 8  *
 9  193.251.241.121  95.295 ms
