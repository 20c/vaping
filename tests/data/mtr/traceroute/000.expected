["204.130.133.130", "38.122.209.113", "154.24.55.105", "154.54.0.225", "154.54.45.18", "154.54.10.34", "193.251.241.121"]
